### PR TITLE
zfs and spl update

### PR DESCRIPTION
**Pull request by nmeum**
Since those two new versions don't appear on the website I just decided to download them from github…

### CODE REVIEW

File: srcpkgs/spl/template [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # Template file for 'spl'
 pkgname=spl
-version=0.6.3
-revision=2
+version=0.6.3-1.3
```
Comment (xtraeme) [inline srcpkgs/spl/template]: This is wrong, version cannot contain dashes.

File: srcpkgs/zfs/template [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'zfs'
 pkgname=zfs
-version=0.6.3
-revision=3
+version=0.6.3-1.3
```
Comment (xtraeme) [inline srcpkgs/zfs/template]: Ditto, version cannot contain dashes.

**Comment by nmeum** [comment]
Well…that's what the version is called…

**Comment by xtraeme** [comment]
Replace the dash with something else or I'll close the PR :-)

File: srcpkgs/spl/template [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+revision=1
 build_style=gnu-configure
 configure_args="--with-config=user --sbindir=/usr/bin"
 short_desc="Solaris Porting Layer -- userland and kernel modules (using DKMS)"
 maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
 license="GPL-2"
 homepage="http://zfsonlinux.org/"
-distfiles="http://archive.zfsonlinux.org/downloads/zfsonlinux/spl/spl-${version}.tar.gz"
-checksum=898aa3912c73420a0082375679e28267a26d348a84561e25898a171785f41fe3
+distfiles="https://github.com/zfsonlinux/${pkgname}/archive/${pkgname}-$(echo $version | sed 's/_/-/').tar.gz"
```
Comment (Gottox) [inline srcpkgs/spl/template]: Use `${version/_/-}` :)

File: srcpkgs/zfs/template [hunk 4]
```diff
@@ -13,8 +13,8 @@ short_desc="Z File System -- userland and kernel modules (using DKMS)"
 maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
 license="CDDL"
 homepage="http://zfsonlinux.org/"
-distfiles="http://archive.zfsonlinux.org/downloads/zfsonlinux/zfs/zfs-${version}.tar.gz"
-checksum=29bcfdd4161521692d2ca2515ad27a9edd9491c9cb5c6fb1e923d8eed28d391b
+distfiles="https://github.com/zfsonlinux/${pkgname}/archive/${pkgname}-$(echo $version | sed 's/_/-/').tar.gz"
```
Comment (Gottox) [inline srcpkgs/zfs/template]: Same here

**Comment by xtraeme** [comment]
Assigned to maintainer.

**Pull request merged**